### ISSUE TITLE
Asking user to re-login every-time the browser is closed

### ISSUE THREAD

**Issue by vanuj** [keycloak/keycloak#23082]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the [latest nightly release](https://github.com/keycloak/keycloak/releases/tag/nightly)

### Area

authentication

### Describe the bug

Hi All, I am working on providing a general functionality where the user closes the browser(without logging out) should be asked to re-login once he opens the browser again.

Steps to reproduce : 
1. Login to client application using keycloak.
2. Navigate through website secured pages(you can verify session cookies are available in browser inspection)
3. Now close the browser (do not logout, simple close the browser)
4. **Now open the browser and go to the application secured page again, it won't ask you to login again.** (Issue)

Ideally, we expect user to login again, I have tried "login-required" and "check-sso" seperately, but it didn't work.

There should be some configuration in keycloak that can be enabled/disabled to force the user to relogin. This will allow admins to make applications more secure depending on their usecase.
This functionality seems to be very general we get these days in websites dealing with user account management.

### Version

^21.0.1

### Expected behavior

1. Some sort of configuration at Keycloak level that can be on/off based on need.
2. Keycloak should ask the user to log in again if the user closes the browser and reopen the website.

### Actual behavior

It doesn't ask user to login again. It simply maintains the session.

### How to Reproduce?

Steps to reproduce : 
1. Login to client application using keycloak.
2. Navigate through website secured pages(you can verify session cookies are available in browser inspection)
3. Now close the browser (do not logout, simple close the browser)
4. **Now open the browser and go to the application secured page again, it won't ask you to login again.** (Issue)

**Comment by douglaspalmer** [comment]
Keycloak already supports this. You need to disable "Remember Me" as described here: https://www.keycloak.org/docs/latest/server_admin/#enabling-remember-me

**Comment by vanuj** [comment]
Hi @douglaspalmer , I already have "Remember Me" disabled in the Keycloak console but the issue is still there. I seem the session is not being cleared in some browsers as we close it. But, the more I investigate it more confusing it becomes. 
After I updated my Chrome to "Version 117.0.5938.132 (Official Build) (arm64)" it started working for me, i.e. once I closed the browser it asked me to log in again. Although with the same Chrome version, it is not working on a few co-worker's laptops.
